PurchasesCoreSwiftTests/Purchasing/ProductInfoExtractorTests.swift

```diff
@@ -181,14 +181,14 @@ class ProductInfoExtractorTests: XCTestCase {
 
             let receivedProductInfo = productInfoExtractor.extractInfo(from: product)
 
-            expect(receivedProductInfo.subscriptionGroup).to(beEmpty())
+            expect(receivedProductInfo.subscriptionGroup).to(beNil())
```

Comment:
The extractor returns `nil` instead of an empty array

Reply:
✅